src/content/docs/en/setup/getting.mdx

```diff
@@ -32,38 +27,29 @@ Ultramarine Linux is distributed in various formats available for download from
   sites other than our website or FOSSTorrents
 </Alert>
 
-### Atomic
+### Bootc Preview
 
-Ultramarine Linux also comes in Atomic versions. Ultramarine Atomic works similarly to Fedora Silverblue, CoreOS, and other OSTree-based operating systems. This variant is made for users who want a stable, immutable system that is easy to recover from.
-We see immutability as the future of Linux systems. Ideally, this will become the standard edition in the future.
+With Ultramarine 41 we are now providing a developer preview of our bootc variant. Ultramarine bootc is based on Fedora bootc, but we intend to switch to our own base container (based on Fedora packages) in the future.
+Bootc. We see this as the future of Linux systems. Ideally, this will become our standard edition in the future.
 
-These builds are currently in beta. If you are planning to test them, please join our [Discord](https://discord.gg/5fdPuxTg5Q) or [Matrix](https://matrix.to/#/#hub:fyralabs.com) community first and let us know. We'll be happy to receive feedback and assist you.
+These builds are not ready for daily use by the average user. If you are planning to test them, please join our [Discord](https://fyralabs.com/discord) or [Matrix](https://fyralabs.com/matrix) community and keep us posted on any bugs you find, from a papercut to a broken system.
 
-[Try Ultramarine Atomic →](https://nightly.link/Ultramarine-Linux/ostree-config/workflows/ci-isos.yaml/main?preview)
+[Try Ultramarine bootc →](todo)

```

Comment:
fix your todo

Reply:
link to bootc repo, i will add install instructions to the readme, currently the only way to install is either rebasing from ublue (not fedora atomic yet) or deploying to the disk manually